index.html

```diff
@@ -523,6 +531,116 @@ <h4>Workflow Type</h4>
+            which represents the default language for the <a>Text</a> content of <a>Script Events</a>.
+            This language may be one of the <a>Original Languages</a> or a <a>Translation</a> language.
+            When it represents a <a>Translation</a> language, it may be the final language
+            for which a dubbing or audio description <a>script</a> is being prepared,
+            called the <dfn>Target Recording Language</dfn> or it may be an intermediate, or pivot, language
+            used in the workflow.
+          </p>
+
+          <p>The <a>Default Language</a> is represented in a <a>DAPT Document</a> by the following structure and constraints:
+            <ul>
+              <li>the <code>xml:lang</code> attribute MUST be present on the <code>&lt;tt&gt;</code> element and its value MUST NOT be empty.</li>
+            </ul>
+          </p>
+          <p class="note">All text content in a <a>DAPT Script</a> has a specified language.
+            When multiple languages are used, the <a>Default Language</a> can correspond to the language of the majority of <a>Script Events</a>,
+            to the language being spoken for the longest duration, or to a language arbitrarily chosen by the author.</p>
+          <aside class="example">An <a>Original Language Transcript</a> of dialogue is prepared for a video
+            containing dialogue in Danish and Swedish.
+            The <a>Default Language</a> is set to Danish by setting <code>xml:lang="da"</code> on the <code>&lt;tt&gt;</code> element.
+            <a>Script Events</a> that contain Swedish <a>Text</a> override this by setting
+            <code>xml:lang="sv"</code> on the <code>&lt;p&gt;</code> element.
+            <a>Script Events</a> that contain Danish <a>Text</a> can set the <code>xml:lang</code> attribute
+            or omit it, since the inherited language is the <a>Default Language</a> of the document.
+            In both cases the Script Events' <a>Text</a> objects are <code>&lt;p&gt;</code> elements that represent untranslated
+            content that had an inherent language (in this case dialogue)
+            and therefore set the <a><code>daptm:langSrc</code></a> attribute to their source language,
+            implying that they are in the <a>Original</a> language.</aside>
+        </section>
+
+        <section>
+          <h4>Original Languages</h4>
+          <p>
+            The <dfn data-lt="Original languages list">Original Languages</dfn>,
+            also referred to as "Original languages list",
+            is an optional property of a <a>DAPT Script</a>
+            which represents the languages used in the original content,
+            in other words,
+            the set of languages associated with <a>Text</a> objects in the <a>Original Language Transcript</a>.
+          </p>
+          <p class="note">
+            <a>Text</a> marked as <a>Original</a> can be in a 
```

Comment:
Only minor: text is not `marked` anymore as Original. Text is defined as Original if one of four statements are true. Therefore it might be better to replace `marked` with `defined`.

Reply:
Oh, @cconcolato wanted consistency in the terminology here too, and seemed to be in favour of using "marked as" in https://github.com/w3c/dapt/pull/179#discussion_r1421218985, and even suggested defining something related to this as a term.

Reply:
My understanding of "marked" is that there is additional Markup. But I am fine with any other solution.

Reply:
I'm happy to make a change here, I'm just not sure _which_ change to make!

Reply:
Somehow I can not access the comment from @cconcolato but if in the document "marked as" is consistently used with the meaning of "has the semantics of" I am also fine with keeping it as is.

Reply:
I made a proposal in https://github.com/w3c/dapt/pull/179#discussion_r1428367490. Essentially, I suggest:
- not defining any new term (e.g. marked as)
- stop repeating the semantics of Original or Translation text
- using the term "marked as" and directly refer to Original Text objects or Translation Text objects.

Reply:
> not defining any new term (e.g. marked as)
> using the term "marked as" and directly refer to Original Text objects or Translation Text objects.

@cconcolato Do you mean "**not**  using the term 'marked as'..."? Otherwise fine with the proposals in the three bullet points.

Reply:
@andreastai I modified my comment. I did mean "NOT using".

Reply:
FYI, there is a sentence that uses "mark" (not marked), but it is fine to leave it as is:
>  The combination of the xml:lang and daptm:langSrc attributes are used to mark the text as being original or translated